editor.planx.uk/src/@planx/components/FindProperty/Editor.tsx

```diff
@@ -55,6 +60,44 @@ function FindPropertyComponent(props: Props) {
             />
           </InputRow>
         </ModalSectionContent>
+        <ModalSectionContent>
+          <InputRow>
+            <OptionButton
+              selected={formik.values.allowNewAddresses}
+              onClick={() => {
+                formik.setFieldValue(
+                  "allowNewAddresses",
+                  !formik.values.allowNewAddresses
+                );
+              }}
+            >
+              Allow users to plot new addresses without a UPRN
+            </OptionButton>
+          </InputRow>
+          {formik.values.allowNewAddresses ? (
```

Comment:
Question: this feature is already behind a toggle in the Editor, do we want to additionally wrap in a Feature Flag for good measure?

Also worth noting: all new Editor props (eg `allowNewAddresses`) are optional, meaning this change should _not_ require a content migration/component "update" when merged. It will default to `ExternalPlanningSiteDialog` same as now.

Reply:
Keeping `<ExternalPlanningSiteDialog />` as an option is a great idea 👍 

I'd love to understand how this change is going to work for Uniform submissions, in particular Southwark who have a different address set up to everybody else.

On this point - I think it's fine to expose this as is? I think it's unlikely someone will toggle this, publish, and not appreciate what it's doing.